tckprocess.adoc

```diff
@@ -0,0 +1,422 @@
+exclusion of that test from certification requirements, and an immediate
+update and release of an official distribution of the TCK including the
+new exclude list. The associated `challenge` issue MUST be closed with
+an `accepted` label to indicate it has been resolved.
+
+[#_rejected_challenges_and_remedy]
+#### Rejected Challenges and Remedy
+
+When a `challenge` issue is rejected, it MUST be closed with a label of
+`invalid` to indicate it has been rejected. The appeal process for
+challenges rejected on technical terms is outlined in Escalation Appeal.
+If, however, an implementer feels the TCK challenge process was not
+followed, an appeal issue MUST be filed with the specification's
+issue tracker using the label `challenge-appeal`. A contributor MUST
+escalate the issue with the MicroProfile Specification Committee via email
+(<microprofile-wg@eclipse.org>). The committee will evaluate the matter
+purely in terms of due process. If the appeal is accepted, the original
+TCK challenge issue will be reopened and a label of `appealed-challenge`
+added, along with a discussion of the appeal decision, and the
+`challenge-appeal` issue with be closed. If the appeal is rejected, the
+`challenge-appeal` issue MUST be closed with a label of `invalid`.
+
+image:tckprocess.png[TCK Process]
+
+[#_excludes]
+## Excludes
+
+Excludes MUST be included in the specification's TCK release in a format that is
+compatible with the testing framework in use so that as the excludes are
+updated, the affected tests are automatically removed from the test
+suite.
+
+[#_improvement]
+## Improvement
+
+Requests for improvement to tests MUST simply be created as issues with
+a label of `enhancement` in the specification's TCK issue
+tracker.
+
+[#_certification_of_compatibility]
+## Certification of Compatibility
+
+MicroProfile is a self-certification ecosystem. If you wish to have your
+implementation listed on the official https://microprofile.io[microprofile.io]
+implementations page for the given specification, a certification
+request as defined in this section is required.
+
+There are additional requirements that MUST be met by any organization
+wishing to use the MicroProfile Platform Compatible logo or
+https://microprofile.io[microprofile.io] website for promotion. Any
+request for certification from an organization not meeting the requirements
+will be held until such time as the requirements are met. See the
+https://microprofile.io/legal/trademark_guidelines/[MicroProfile Trademark Guidelines] for more
+information on the complete steps to obtain  branding usage.
+
+An approved certification request is a statement from the Specification
+contributors you have met the intended TCK requirements and is just one of
+the requirements for logo usage. Logos only apply to the platform specification.
+
+[#_filing_a_certification_request]
+### Filing a Certification Request
+
+Requests to be acknowledged as a certified implementation MUST be filed
```

Comment:
Can we spell out the repo location https://github.com/microprofile/microprofile-wg

Reply:
We discussed on the call. For umbrella release certification request, it is better to be filed under https://github.com/eclipse/microprofile